pkg/kritis/review/validating_transport.go

```diff
@@ -69,19 +69,20 @@ func (avt *AttestorValidatingTransport) GetValidatedAttestations(image string) (
 		return nil, err
 	}
 	for _, a := range attestations {
-		// TODO(acamadeo): Temporarily hardcoding this to work only for PGP Attestations.
-		signature := a.PgpAttestation.Signature.Signature
-		decodedSig, err := base64.StdEncoding.DecodeString(string(signature))
-		if err != nil {
-			glog.Infof("Cannot base64 decode signature: %v", err)
-			continue
-		}
-		keyId := a.PgpAttestation.Signature.PublicKeyId
-		if err = host.VerifyAttestationSignature(keys[keyId], string(decodedSig)); err != nil {
-			glog.Infof("Could not find or verify attestation for attestor %s: %s", keyId, err.Error())
-			continue
+		for _, rawSig := range a.Signatures {
```

Comment:
Tracing through the code, it seems like a.Signatures can only ever contain a single signature.  I suggest changing the RawAttestation struct to only have a single RawSignature.

There will be some other consequences to this, so let's just talk it over tomorrow before making the changes.

Reply:
Done.